src/server/components/dashboard/annualReview/helpers.ts

```diff
@@ -22,57 +22,59 @@ function addSixMonths(date: number | string | Date): Date {
   return newDate;
 }
 
-export function getAnnualReviewYear({
-  day,
-  month,
-  lastAnnualReview,
+export function calculateValidDate({
+  userEnteredDate,
```

Comment:
This is a good point. I have a rule of thumb that if an argument has more than 3-4 arguments it makes sense to pass it down as a single object instead of multiple arguments for readability.

At some point this function did have multiple arguments but was refactored to have 2, so the object is now unnecessary.